list/list.js

```diff
@@ -590,44 +599,46 @@ var DefineList = Construct.extend("DefineList",
 		// pass `key` to only log the matching event, e.g: `list.log("add")`
 		log: function(key) {
 			//!steal-remove-start
-			var instance = this;
+			if(process.env.NODE_ENV !== 'production') {
```

Comment:
Might it have been easier to have log be a no-op here and at the bottom of this file:

```js
//!steal-remove...
if(process.env...){
  DefineList.prototype.log = function(){ ... }
}
//!steal-remove..
```